libraries/chain/asset_evaluator.cpp

```diff
@@ -450,21 +461,29 @@ static void update_bitasset_object_options(
       }
    }
 
+   const auto old_feed_price = bdo.current_feed.settlement_price;
    if( should_update_feeds )
       bdo.update_median_feeds(db.head_block_time());
+
+   return old_feed_price != bdo.current_feed.settlement_price;
 }
 
 void_result asset_update_bitasset_evaluator::do_apply(const asset_update_bitasset_operation& op)
 {
    try
    {
       auto& db_conn = db();
+      const fc::time_point_sec& next_maint_time = db_conn.get_dynamic_global_properties().next_maintenance_time;
       const auto& asset_being_updated = (*asset_to_update);
+      bool price_changed = false;
 
-      db_conn.modify(*bitasset_to_update, [&op, &asset_being_updated, &db_conn](asset_bitasset_data_object& bdo) {
-         update_bitasset_object_options(op, db_conn, bdo, asset_being_updated);
+      db().modify(*bitasset_to_update, [&op, &asset_being_updated, &price_changed, &next_maint_time, &db_conn](asset_bitasset_data_object& bdo) {
```

Comment:
Changed to `db()` here, so perhaps still have conflicts.

I was thinking that we can get `next_maint_time` inside the function.

Reply:
Fixed